fdbclient/include/fdbclient/CommitProxyInterface.h

```diff
@@ -750,6 +754,31 @@ struct GlobalConfigRefreshRequest {
 	}
 };
 
+struct SetThrottledShardReply {
+	constexpr static FileIdentifier file_identifier = 2828140;
+
+	SetThrottledShardReply() {}
+
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar);
+	}
+};
+
+struct SetThrottledShardRequest {
```

Comment:
I think this request should include a expiration period so that we can clear them at CP.